lua/core/clock.lua

```diff
@@ -13,10 +20,16 @@ end
 local SLEEP = 0
 local SYNC = 1
 
+--- suspend execution of a coroutine for s seconds;
+-- requires being called from within a coroutine launched with clock.run.
+-- @tparam float s : seconds
```

Comment:
is the colon part of ldoc syntax? also why there are semicolons in some function descriptions?

Reply:
Not sure (was copying from other norns libraries). I will do some checking/testing and adjust as needed.

Reply:
A period effective ends the description section. A semicolon let's you continue the description with a new comment line without it looking like a run-on sentence.

Reply:
cool, then can you also make sure that the style is consistent with other lua code, i.e. clock.set_source doesn't have period in this change.

Reply:
added periods

Reply:
what about colons?

Reply:
As I understand, the `param` or `tparam` tags can have a colon to separate the param description 
![Screen Shot 2019-07-03 at 12 55 25 AM](https://user-images.githubusercontent.com/1909661/60566693-4a785500-9d2d-11e9-8c3d-64a12fd06735.png)

Reply:
Looks like a colon could be also used instead of a semicolon to separate the main description text.

Reply:
added additional descriptive text for params. Not sure how best to describe "beats"

Reply:
it's a bit awkward name, it's kinda the next fraction of a beat at which we'd like to resume the coroutine (need better wording for the sentence). documentation should mention that it can be more than 1.

Reply:
is this any better?
```
--- suspend execution until the next fraction of a beat is reached in time;
-- must be called from within a coroutine started with clock.run.
-- @tparam float beats : Next fraction of a beat at which the coroutine will be resumed. May be a value larger than 1.
```